tools/quake3/common/vfs.c

```diff
@@ -145,7 +145,7 @@ static void vfsInitPakFile( const char *filename ){
 		g_pakFiles = g_slist_append( g_pakFiles, file );
 
 		vfsFixDOSName( filename_inzip );
-		g_strdown( filename_inzip );
+		g_ascii_strdown( filename_inzip, -1 );//-1 null terminated string
```

Comment:
The above `unzGetCurrentFileInfo` does not guarantee null terminated.

See: https://github.com/ec-/Quake3e/blob/master/code/qcommon/files.c#L2106